lib/finance.ts

```diff
@@ -0,0 +1,66 @@
+import { IndexHistoryPoint } from '../types/finance';
+
+export const TRADING_DAYS_PER_YEAR = 252;
+
+export function computeChangePct(current?: number | null, base?: number | null) {
+  if (current == null || base == null || base === 0) return null;
+
+  return (current - base) / base;
+}
```

Comment:
```suggestion
export const computeChangePct = (current?: number | null, base?: number | null) => (current - base) / base;
```
有问题会计算出 `NaN`，外面也能正常判断：https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NaN